youtube_dl/downloader/external.py

```diff
@@ -192,15 +195,19 @@ def supports(cls, info_dict):
     def available(cls):
         return FFmpegPostProcessor().available
 
+    def get_execname(self):
+        return self.execname
```

Comment:
>How could it work otherwise? `_call_downloader()` instantiates an instance of `FFmpegPostProcessor`, and only at that time is a decision made on the executable name.

It should instantiate `FFmpegPostProcessor` on it's own like `available` do. It may accept `info_dict` as input parameter.

>Meaning as an `@property`? OK.

Meaning both this and implementation.